server/www/templates/activity/time-chart.html

```diff
@@ -0,0 +1,30 @@
+<div class="timeline-view">
+    <fieldset class="custom-fieldset init" dropdown>
+        <legend class="dropdown-trigger init" data-toggle="dropdown-menu" dropdown-toggle>
+            <span class="legend-label" translate="graph.compare">Compare...</span>
+            <svg class="iconic chevron">
+                <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="../../img/iconic-sprite.svg#chevron-bottom"></use>
+            </svg>
+        </legend>
+
+        <div class="dropdown-menu init" dropdown-menu>
+            <div class="form-field radio" ng-repeat="(key, value) in groupByOptions">
+                <label>
+                    <input type="radio" name="groupBy" value="{{key}}" ng-model="$parent.groupBy.value" ng-change="reload()"> {{::value | translate}}
```

Comment:
@willdoran @jasonmule any idea why these radio inputs don't get populated with the correct initial value?
I can sit there reloading and sometimes it works, others it doesnt. So confused.